### PR TITLE
[Fix](doc) Fix word spelling errors

### PR DESCRIPTION
**Pull request by xingyingone**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xingyingone** [comment]
run build all

**Comment by LemonLiTree** [comment]
run buildall

**Review by LemonLiTree — APPROVED** [review]
LGTM

File: docs/zh-CN/docs/admin-manual/workload-group.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ set workload_group = 'g2';
 ```
 session变量`workload_group`优先于 user property `default_workload_group`, 在`workload_group`为空时，查询将绑定到`default_workload_group`, 在session变量`workload_group`不为空时，查询将绑定到`workload_group`。
 
-如果是非admin用户，需要先执行[SHOW-WORKLOAD-GROUPS](../sql-manual/sql-reference/Show-Statements/SHOW-WORKLOAD-GROUPS.md) 确认下当前用户能否看到该workload group，不能看到的workload group可能不存在或者当前用户没有权限，执行查询时会报错。给worklaod group授权参考：[grant语句](../sql-manual/sql-reference/Account-Management-Statements/GRANT.md)。
+如果是非admin用户，需要先执行[SHOW-WORKLOAD-GROUPS](../sql-manual/sql-reference/Show-Statements/SHOW-WORKLOAD-GROUPS.md) 确认下当前用户能否看到该workload group，不能看到的workload group可能不存在或者当前用户没有权限，执行查询时会报错。给workload group授权参考：[grant语句](../sql-manual/sql-reference/Account-Management-Statements/GRANT.md)。
```
Comment (zy-kkk) [inline docs/zh-CN/docs/admin-manual/workload-group.md:147]: Can you modify it again so that there is a space between English and Chinese?

Comment (xingyingone) [reply docs/zh-CN/docs/admin-manual/workload-group.md:147]: ok

**Comment by xingyingone** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Pull request merged**